Travis fails on:

```
"  Running setup.py bdist_wheel for pysqlite: finished with status 'error'", 
"  Complete output from command /edx/app/ecommerce/venvs/ecommerce/bin/python2.7 -u -c \"import setuptools, tokenize;__file__='/tmp/pip-build-FwAYSs/pysqlite/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\\r\\n', '\\n');f.close();exec(compile(code, __file__, 'exec'))\" bdist_wheel -d /tmp/tmpVkEcs3pip-wheel- --python-tag cp27:", 
"  running bdist_wheel", 
"  running build", 
"  running build_py", 
"  creating build", 
"  creating build/lib.linux-x86_64-2.7", 
"  creating build/lib.linux-x86_64-2.7/pysqlite2", 
"  copying lib/__init__.py -> build/lib.linux-x86_64-2.7/pysqlite2", 
"  copying lib/dump.py -> build/lib.linux-x86_64-2.7/pysqlite2", 
"  copying lib/dbapi2.py -> build/lib.linux-x86_64-2.7/pysqlite2", 
"  creating build/lib.linux-x86_64-2.7/pysqlite2/test", 
"  copying lib/test/factory.py -> build/lib.linux-x86_64-2.7/pysqlite2/test", 
"  copying lib/test/__init__.py -> build/lib.linux-x86_64-2.7/pysqlite2/test", 
"  copying lib/test/dbapi.py -> build/lib.linux-x86_64-2.7/pysqlite2/test", 
"  copying lib/test/dump.py -> build/lib.linux-x86_64-2.7/pysqlite2/test", 
"  copying lib/test/userfunctions.py -> build/lib.linux-x86_64-2.7/pysqlite2/test", 
"  copying lib/test/regression.py -> build/lib.linux-x86_64-2.7/pysqlite2/test", 
"  copying lib/test/hooks.py -> build/lib.linux-x86_64-2.7/pysqlite2/test", 
"  copying lib/test/transactions.py -> build/lib.linux-x86_64-2.7/pysqlite2/test", 
"  copying lib/test/types.py -> build/lib.linux-x86_64-2.7/pysqlite2/test", 
"  running build_ext", 
"  building 'pysqlite2._sqlite' extension", 
"  creating build/temp.linux-x86_64-2.7", 
"  creating build/temp.linux-x86_64-2.7/src", 
"  x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fno-strict-aliasing -Wdate-time -D_FORTIFY_SOURCE=2 -g -fstack-protector-strong -Wformat -Werror=format-security -fPIC -DMODULE_NAME=\"pysqlite2.dbapi2\" -Ish: -I1: -Ipkg-config: -Inot -Ifound -I/usr/include/python2.7 -c src/module.c -o build/temp.linux-x86_64-2.7/src/module.o", 
"  In file included from src/module.c:24:0:", 
"  src/connection.h:33:21: fatal error: sqlite3.h: No such file or directory", 
"  compilation terminated.", 
"  error: command 'x86_64-linux-gnu-gcc' failed with exit status 1"
```

Ie, the usual "something utterly unrelated to this or any other recent change" and we're once again left wondering how the tests *ever* passed....

--- a/playbooks/roles/nginx/templates/edx/app/nginx/sites-available/lms-common-settings.j2
+++ b/playbooks/roles/nginx/templates/edx/app/nginx/sites-available/lms-common-settings.j2
@@ -224,6 +224,10 @@ location ~ ^{{ EDXAPP_MEDIA_URL }}/(?P<file>.*) {
 
   {% include "extra_locations_lms.j2" ignore missing %}
 
+  {% if nginx_lms_extra_locations != "" %}
+    {{ nginx_lms_extra_locations }}
+  {% endif %}
+
   {% if EDXAPP_XBLOCK_SETTINGS.get('ScormXBlock', False) %}
     # asterisk needed so it doesn't break if the file doesn't exist
     include {{nginx_app_dir}}/includes/scorm_extra_locations_lms*;

